interaction model/schema.json

```diff
@@ -31,6 +31,12 @@
       ]
     },
     {
+      "intent": "AMAZON.YesIntent"
```

Comment:
Nit - could you please move these to the top of the schema, where the other `AMAZON.*` intents are?  I've been trying to keep the simple Amazon intents all grouped at the top.